submissions/ArthurGorbenko/Friend's app/js/script.js

```diff
@@ -0,0 +1,282 @@
+};
+
+const refreshDMWindow = () => {
+  CONSTS.FORM.classList.remove("visually-hidden");
+  CONSTS.ALERT_MESSAGE.classList.add("visually-hidden");
+  CONSTS.TEXT_AREA.value = "";
+};
+
+let menuToggled = false;
+
+const moveMenu = () => {
+  let position = 0;
+  let middleScreen = Math.floor(
+    (window.innerWidth - CONSTS.WRAPPER_MENU.clientWidth) / 2
+  );
+  let id = setInterval(relocate, 1);
+  function relocate() {
+    if (position === middleScreen) {
+      clearInterval(id);
+    } else {
+      position++;
+      CONSTS.WRAPPER_MENU.style.left = position + "px";
+    }
+  }
+  menuToggled = true;
+};
+
+const moveContent = () => {
+  CONSTS.LIST_OF_CARDS.classList.toggle("move-bottom");
+  CONSTS.HAMBURGER.classList.toggle("is-active");
+  if (!menuToggled) {
+    moveMenu();
+  } else {
+    CONSTS.WRAPPER_MENU.style.left = "600px";
+    menuToggled = false;
+  }
+};
+
+const writeLetter = ({ target }) => {
+  let mediaQuery = window.matchMedia(MOBILE_RESOLUTION);
+  if (CONSTS.FORM.classList.contains("visually-hidden")) {
+    refreshDMWindow();
+  }
+  if (target.classList.contains("sendMessage")) {
+    CONSTS.DIRECT_MESSAGE_FORM.classList.remove("hide-direct-message");
+    CONSTS.EMAIL_FIELD.value = target.parentElement.dataset.email;
+    if (mediaQuery.matches) {
+      CONSTS.HAMBURGER.classList.remove("visually-hidden");
+      window.scrollTo({ top: 0, behavior: "smooth" });
+      if (!menuToggled) {
+        moveContent();
+        menuToggled = true;
+      }
+    }
+  }
+};
+
+const messageSend = () => {
+  CONSTS.FORM.classList.add("visually-hidden");
+  CONSTS.ALERT_MESSAGE.classList.remove("visually-hidden");
+};
+
+const addLiElements = parent => {
+  let props = {
+    name: document.createElement("li"),
+    age: document.createElement("li"),
+    phone: document.createElement("li")
+  };
+  for (prop in props) {
+    props[prop].classList.add("infoFriend");
+    parent.append(props[prop]);
+  }
+  return props;
+};
+
+const createBlock = (tag, classList) => {
+  element = document.createElement(tag);
+  element.classList.add(...classList);
+  return element;
+};
+
+const createCard = user => {
+  let li = createBlock("li", ["card"]);
+  li.dataset.email = user.email;
+
+  let img = createBlock("img", ["photo"]);
+  img.src = user.picture.large;
+
+  let listInfo = createBlock("ul", ["listInfo"]);
+
+  let sendLetter = createBlock("input", ["button", "sendMessage"]);
+  sendLetter.type = "button";
+  sendLetter.value = "Send letter";
+
+  [img, listInfo, sendLetter].forEach(el => li.append(el));
+
+  let properties = addLiElements(listInfo);
+
+  properties.age.innerHTML = `<strong>Age : </strong> ${user.dob.age}`;
+  properties.name.innerHTML = `<strong>Name : </strong>
+    ${letterToUpperCase(user.name.first)}
+    ${letterToUpperCase(user.name.last)}`;
+  properties.phone.innerHTML = `<strong>Phone : </strong> ${user.phone}`;
+
+  CONSTS.LIST_OF_CARDS.append(li);
+};
+
+const makeDarkBackground = () => {
```

Comment:
function name is makeDarkBackground, but inside you toggle dark clases. Maybe this function is 
toogleAppTheme or something like this.

Reply:
Renamed.